NEWS.md

```diff
@@ -33,7 +33,9 @@ New library functions
 * Two argument methods `findmax(f, domain)`, `argmax(f, domain)` and the corresponding `min` versions ([#27613]).
 * `isunordered(x)` returns true if `x` is value that is normally unordered, such as `NaN` or `missing`.
 * New macro `Base.@invokelatest f(args...; kwargs...)` provides a convenient way to call `Base.invokelatest(f, args...; kwargs...)` ([#37971])
-* New macro `Base.@invoke f(arg1::T1, arg2::T2; kwargs...)` provides an easier syntax to call `invoke(f, Tuple{T1,T2}, arg1, arg2; kwargs...)` ([#38438])
+* New macro `Base.@invoke f(arg1::T1, arg2::T2; kwargs...)` provides an easier syntax to call `invoke(f, Tuple{T1,T2}; kwargs...)` ([#38438])
```

Comment:
This line shouldn't have been changed.

Reply:
Sir The new line was causing a conflict with my branch so I just kept the original JuliaLang/Julia News.md file.

Reply:
You may need to do a rebase to fix the conflict?

Reply:
Sir, but now the conflict is already fixed and I have also added the new functions in the respective files.